app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  scope :recent_five, ->(postid) { where(postid == :post_id).limit(5).order(created_at: :desc) }
```

Comment:
- [ ] Instead of using custom queries that you define in your class using `scope method`, I'll suggest you to define simple method. Here, you have to implement a method that updates the comment counter. take a look here:
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```